dune-project

```
@@ -0,0 +1,3 @@
+(lang dune 3.17)
+(using coq 0.8)
+(name skylabs-fm)
```

```suggestion
(lang dune 3.17)
(name workspace)
```
I think this should be sufficient, and a more suitable project name (might conflict otherwise).